packages/core/src/docdb/wizards/createClusterWizard.ts

```diff
@@ -0,0 +1,198 @@
+                        label: localize('AWS.docdb.createCluster.storage.encrypted', 'Encrypt'),
+                        description: '(recommended)',
+                        data: true,
+                    },
+                    {
+                        label: localize('AWS.docdb.createCluster.storage.notEncrypted', "Don't encrypt"),
+                        data: false,
+                    },
+                ],
+                {
+                    step: 5,
+                    title: localize('AWS.docdb.createCluster.storageEncrypted.prompt', 'Specify storage encryption'),
+                    buttons: createCommonButtons(DocDBHelpUrl),
+                }
+            )
+        )
+
+        form.DBInstanceCount.bindPrompter(() =>
+            createQuickPick(instanceCountItems(), {
+                step: 6,
+                title: localize('AWS.docdb.createCluster.dbInstanceCount.prompt', 'Number of instances'),
+                buttons: createCommonButtons(DocDBHelpUrl),
+            })
+        )
+
+        form.DBInstanceClass.bindPrompter(
+            async (state) => await createInstanceClassPrompter(this.client, state.EngineVersion!)
+        )
+
+        form.DBInstanceCount.setDefault(3)
+        form.DBInstanceClass.setDefault('db.t3.medium')
+
+        return this
+    }
+}
+
+async function createEngineVersionPrompter(docdbClient: DocumentDBClient) {
+    const versions = await docdbClient.listEngineVersions()
+    // sort in descending order
+    versions.sort((a, b) => b.EngineVersion!.localeCompare(a.EngineVersion!))
+
+    const items: DataQuickPickItem<string>[] = versions.map((v) => {
+        return {
+            label: v.EngineVersion ?? '',
+            data: v.EngineVersion,
+        }
+    })
+
+    if (items.length === 0) {
+        items.push({ label: '5.0.0 (default)', data: '5.0.0' })
+    }
+
+    items[0].picked = true
+
+    return createQuickPick(items, {
+        step: 2,
+        title: localize('AWS.docdb.createCluster.engineVersion.prompt', 'Select engine version'),
+        buttons: createCommonButtons(DocDBHelpUrl),
+    })
+}
+
+async function createInstanceClassPrompter(docdbClient: DocumentDBClient, engineVersion: string) {
+    const options = await docdbClient.listInstanceClassOptions(engineVersion, DBStorageType.Standard)
+
+    const items: DataQuickPickItem<string>[] = options.map((option) => {
+        return {
+            data: option.DBInstanceClass,
+            label: option.DBInstanceClass ?? '(unknown)',
+            description: undefined,
+            detail: undefined,
+        }
+    })
+
+    if (items.length === 0) {
+        return new SkipPrompter('db.t3.medium')
+    }
+
+    return createQuickPick(items, {
+        step: 7,
+        title: localize('AWS.docdb.createInstance.instanceClass.prompt', 'Select instance class'),
+        buttons: createCommonButtons(DocDBHelpUrl),
+    })
+}
+
+function instanceCountItems(max: number = 16): DataQuickPickItem<number>[] {
```

Comment:
I recall the max instance count being defined in another file as well, maybe createInstance? Does it make sense to make that number a const and share that here as well?

Reply:
refactored to const